### ISSUE TITLE
Pikaday tab order skips the Month and Year drop downs.

### ISSUE THREAD

**Issue by imcbride** [middlebury/frontend#144]
Steps to reproduce:
1. Go to https://www.middlebury.edu/institute/events
2. Tab to the previous and next buttons on the date picker, which skips over the month and year dropdowns.

For WCAG 2.4.3, we should decide if this is the intended behavior. Additionally, if you remove the `tabindex="-1"` that is placed on the selects, the tab order works logically, but there is no focus applied to the control, so it's hard for the user to know.
